ansible/roles/kong-api/defaults/main.yml

```diff
@@ -4494,3 +4494,19 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+  - name: v2UserBulkUpload
```

Comment:
Change the API name to UserBulkUploadv2, let the APIs following same naming convention

Reply:
DONE